dashboard-for-web/articles/web-dashboard-designer-mode/designing-dashboard-items/pivot/conditional-formatting.md

```diff
@@ -4,36 +4,88 @@ author: Natalia Kazakova
+
+## Create and Edit a Format Rule   
+
+You can create and edit format rules in the **Conditional Formatting** section that is located in the following places:
+
+* The dashboard item's [Options](../../ui-elements/dashboard-item-menu.md) menu
+
+*  [data item menu](../../ui-elements/data-item-menu.md)
+
+Refer to the following topic for information on how to create and edit format rules: [Conditional Formatting in Web Dashboard](../../appearance-customization/conditional-formatting.md).
+
+## Pivot-Specific Format Condition Settings
 
-You can use [hidden measures](../../binding-dashboard-items-to-data/hidden-data-items.md) to specify a condition used to apply formatting to visible values. New appearance settings are applied to data cells corresponding to a row/column intersection. You can set the intersection of the row and column manually or use predefined settings.
+New appearance settings are applied to data cells that correspond to a row/column intersection. You can set a new intersection of the row and column or use predefined settings.
 
-## Create and Edit Format Rules
-To create a new format rule for the Pivot's dimension/measure, select the required data item by whose values a format condition will be calculated, open its menu and go to the **Conditional Formatting** section. Click "+" to add a new rule.
+Note the following specifics:
 
-![wdd-pivot-cf-add-rule](../../../../images/img126058.png)
+1. The dashboard does not calculate format rules in a pivot for percentage values on multiple levels. In this case, "All Levels" intersection mode is not available.
+2. If you create a new format rule for a dimension from the Columns/Rows section, the corresponding format condition dialog does not contain any Pivot-specific settings.
 
-Then, specify the data item to which conditional formatting is applied using the **Apply to** combo box and select the condition type.
+The format rule's **Miscellaneous** section contains pivot-specific options:
 
-![wdd-pivot-cf-select-rule](../../../../images/img126060.png)
+![](../../../../images/web-cf-pivot-miscellaneous.png)
 
-Depending on the selected format condition, the menu used to create a format rule for Pivot contains different settings. For example, the image below displays the _Top/Bottom_ format condition menu. Here, you need to specify the number of values to be displayed and select a format rule style.
+| Option | Description |
+| --|--|
+| **Enabled** | Enables/ Disables the current format rule. |
+| **Intersection Mode** | Specifies the level on which to apply conditional formatting to pivot cells. |  
+| **Intersection Row/Column Dimension**  | Applies the format rule to the specified row/column dimension, if you select the _Specific Level_ as the intersection mode.|
+| **Apply to Row/Column** | Specifies whether to apply the formatting to the entire Pivot's row/column.|
```

Comment:
Do you mean:

`Specifies whether to apply the formatting to the Pivot item's entire row/column.`